doc/_static/custom_scripts.js

```diff
@@ -0,0 +1,162 @@
+// Custom scripts for the Spyder docs site
+
+
+;(function () {
+
+    'use strict';
+
+    // Set the active image for the tour based on the element's class
+    function setActiveTourImage(activeElement) {
+        var activeClass = "tour-screenshot-active";
+        var classNames = activeElement.node.className.baseVal.split(" ");
+        for (var i = 0; i < classNames.length; i++) {
+            var imageToActivate = document.getElementById(classNames[i]);
+            if (imageToActivate) break;
+        };
+        var imageToDeactivate = document.getElementsByClassName(activeClass)[0];
+        imageToDeactivate.classList.remove(activeClass);
+        imageToActivate.classList.add(activeClass);
+    };
+
+    // Interactive tour driver
+    var driver = new Driver({
+        animate: false,
+        opacity: 0.1,
+        padding: 0,
+        allowClose: false,
+        onHighlightStarted: setActiveTourImage,
+    });
+
+    // Event handler to start tour
+    function startTour() {
+        driver.start();
+    };
+
+    // Interactive tour of Spyder for Quickstart using Driver
+    function setupTourDriver() {
+        driver.defineSteps([
+          {
+            element: "#introduction-rect",
+            popover: {
+              title: "Spyder",
+              description: 'Spyder is a powerful scientific IDE for Python. We will guide you through some of its most important features.',
+              position: "right",
+            },
+          },
+          {
+            element: "#toolbar-rect",
+            popover: {
+              title: "Toolbar",
+              description: 'The toolbar allows you to quickly access some of the most common commands in Spyder.',
+              position: "bottom",
+            },
+          },
+          {
+            element: "#statusbar-rect",
+            popover: {
+              title: "Status Bar",
+              description: 'The status bar shows your current Python environment, git branch, memory usage and various attributes of the currently active file.',
+              position: "top",
+            },
+          },
+          {
+            element: "#options-menu-rect",
+            popover: {
+              title: "Options Menu",
+              description: 'You can display the options menu by clicking the "hamburger" icon at the top right of the pane. It contains useful settings and actions relevant to the pane.',
```

Comment:
```suggestion
              description: 'You can display the options menu on every pane by clicking the "hamburger" icon at the top right of the pane. It contains useful settings and actions relevant to the pane.',
```

Just emphasizing that that menu is present in all panes.

Reply:
```suggestion
              description: 'You can display each pane's options menu by clicking the "hamburger" icon at the top right. It contains useful settings and actions relevant to the pane.',
```